client/extensions/woocommerce/components/delta/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+
+export default class Delta extends Component {
+
+	static propTypes = {
+		className: PropTypes.string,
+		isIncreaseFavorable: PropTypes.bool,
+		sinceLabel: PropTypes.string,
+		decimalValue: PropTypes.number.isRequired,
+	};
+
+	static defaultProps = {
+		isIncreaseFavorable: true,
+	};
+
+	render() {
+		const { className, isIncreaseFavorable, sinceLabel, decimalValue } = this.props;
+		const deltaClasses = classnames( 'delta', className );
+		const arrowClasses = classnames( 'delta__icon', {
+			'is-good': ( decimalValue > 0 && isIncreaseFavorable ) || ( decimalValue < 0 && ! isIncreaseFavorable ),
+			'is-bad': ( decimalValue < 0 && isIncreaseFavorable ) || ( decimalValue > 0 && ! isIncreaseFavorable ),
```

Comment:
@westi has gotten the deltas computed in the back end, including classes such as `is-favorable` and `is-increase`.

On a component design note: My philosophy is to remove as much logic out of the presentational components as possible. This allows for maximum reuse. Could this component simply accept the classes from the backend and use apply them as is?